src/Webapi/Webapi__Fetch.res

```diff
@@ -336,6 +336,7 @@ module RequestInit = {
     ~integrity: string="",
     ~keepalive: option<bool>=?,
     ~signal: option<signal>=?,
+    ()
```

Comment:
Pretty sure I left these `()` out because it was easier to write and wasn't necessary to make the output work, this is fine